Source/Initialization/PlasmaInjector.cpp

```diff
@@ -566,17 +567,17 @@ void PlasmaInjector::parseMomentum (ParmParse& pp)
             amrex::Abort(direction.c_str());
         }
         // Construct InjectorMomentum with InjectorMomentumBoltzmann.
-        h_inj_mom.reset(new InjectorMomentum((InjectorMomentumBoltzmann*)nullptr, theta, beta, dir));
+        h_inj_mom.reset(new InjectorMomentum((InjectorMomentumBoltzmann*)nullptr, getTemp, beta, dir));
     } else if (mom_dist_s == "maxwell_juttner"){
         Real beta = 0.;
-        Real theta = 10.;
         int dir = 0;
+        h_mom_temp = std::make_unique<TemperatureProperties>(pp);
+        GetTemperature getTemp(*h_mom_temp.get());
```

Comment:
Does this work?
```suggestion
        GetTemperature getTemp(TemperatureProperties(pp));
```
discussed: or does `h_mom_temp` have to stay alive when we execute the parser? (would be confusing, b/c that's a host object)

Reply:
Alternatively if you want to keep  the `const&` API:
```suggestion
        mom_temp = TemperatureProperties(pp);
        GetTemperature getTemp(mom_temp);
```

Reply:
Shares some state with `TemperatureProperties::m_ptr_temperature_parser` from `h_mom_temp`, so we need to leave it as is.